app/services/encrypted_key_maker.rb

```diff
@@ -59,8 +59,12 @@ def make_kms(user_access_key)
   end
 
   def unlock_kms(user_access_key, encryption_key)
-    ciphertext = user_access_key.xor(decode(encryption_key))
-    user_access_key.unlock(aws_client.decrypt(ciphertext_blob: ciphertext).plaintext)
+    ciphertext = user_access_key.xor(decode(encryption_key)).sub(KEY_TYPE[:KMS], '')
+    begin
```

Comment:
top-level rescue makes sense. It doesn't look like we do anything with the error message. If we want to keep track of how often people enter a wrong password, then adding the message would be useful.